### PR TITLE
add yolov5 voc training

### PR DESCRIPTION
**Pull request by Nioolek**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Add yolov5 voc training.
The map result of yolov5m for vocdataset has reproduced.

The yolov5 training command store at:
https://github.com/ultralytics/yolov5/blob/v6.1/data/hyps/hyp.VOC.yaml#L3

## Modification

Add:
- `configs/yolov5/voc/yolov5_m-v61_syncbn_fast_8xb16-50e_voc.py`
- `configs/yolov5/voc/yolov5_s-v61_syncbn_fast_8xb16-50e_voc.py`
- `mmyolo/datasets/yolov5_voc.py`

TODO:
Make YOLOv5 BatchShapePolicy as a base class.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDetection or MMClassification.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmyolo/datasets/yolov5_voc.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        Get meta information of dataset.
+
+        Returns:
+            dict: meta information collected from ``BaseDataset.METAINFO``,
+            annotation file and metainfo argument during instantiation.
+        """
+        metainfo = copy.deepcopy(self._metainfo)
+        if 'DATASET_TYPE' in metainfo:
+            del metainfo['DATASET_TYPE']
+        return metainfo
```
Comment (PeterH0323) [inline mmyolo/datasets/yolov5_voc.py:73]:
```suggestion
        return metainfo

```

Comment (PeterH0323) [reply mmyolo/datasets/yolov5_voc.py:73]: Missing EOF 😄

**Comment by PeterH0323** [comment]
Can we add a config in `config/_base_/dataset` for different dataset, so the user can switch dataset COCO or VOC format dataset very easy and the develop can use it easily at the same time. Just like mmdet ?

image

**Comment by hhaAndroid** [comment]
@Nioolek We need to align yolov5 official results.

**Pull request closed without merging**